documentation/options.md

```diff
@@ -0,0 +1,42 @@
+Plugin options
+===============
+
+You can specify a few options to the plugin:
+
+```JavaScript
+new RailsManifestPlugin({
+  fileName: 'manifest.json',
+  writeToFileEmit: false,
+  extraneous: null,
+  mapAssetPath: (requirePath) => requirePath
+});
+```
+
+### fileName
+
+This is the output file that will be created after each change. The default is `manifest.json`.
+
+### writeToFileEmit
+
+When using this plugin with [webpack-dev-plugin](#) manifest will not be written out to the filesystem, but to in-memory filesystem of the webpack-dev-plugin.
```

Comment:
Suggesting these changes:
webpack-dev-plugin**, the** manifest **file** will not be written **to a file**, but to **an** in-memory filesystem **used by** webpack-dev-plugin.